libs/application/templates/signature-collection/parliamentary-list-signing/src/forms/Done.ts

```diff
@@ -32,7 +32,12 @@ export const Done: Form = buildForm({
         buildMultiField({
           id: 'doneScreen',
           title: m.listSigned,
-          description: m.listSignedDescription,
+          description: (application: Application) => ({
+            ...m.listSignedDescription,
+            values: {
+              name: (application.answers.list as any).name,
```

Comment:
If you are using application answers, can you not type them with something like:

```typescript
import { infer as zinfer } from 'zod'
type Answers = zinfer<typeof dataSchema>
```
and then cast your application answers to that type :D